woocommerce/woocommerce-android
# [Deposit Summary] Removes feature flag

Closes: #10140


### Description

Makes the feature available in the release builds

### Testing instructions

* Install release build
* Notice Deposit summary in More -> Payments
